ratelimitbackend/admin.py

```diff
@@ -1,5 +1,4 @@
-from django.contrib.admin import *  # noqa
```

Comment:
I don't think such transitive imports are a best practice, but if you insist, I'll revert this and try to hide the flake8 error instead.